.github/workflows/ci.yml

```diff
@@ -170,6 +170,33 @@ jobs:
               '\.py$|.github/workflows/|^Dockerfile|^scripts'
           echo "::set-output name=count::$?"
         id: pyfiles
+
+  test-openapi-client-generation:
+    name: "Test OpenAPI client generation"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+      - name: "Check spec file change"
+        run: |
+          URL="https://raw.githubusercontent.com/apache/airflow/master/airflow/api_connexion/openapi/v1.yaml"
```

Comment:
I think comparing to master is not a good idea. It's better to get list of changed files from the current commit. See `pyfiles` above. There is a ready to use script that you can use for that. Something like this should work: 

```
 ./scripts/ci/ci_count_changed_files.sh ${GITHUB_SHA} 'airflow/api_connexion/openapi/v1\.yaml'
echo "::set-output name=count::$?"
```